detector/test_helper.go

```diff
@@ -0,0 +1,48 @@
+package detector
+
+import (
+	"reflect"
+
+	"github.com/hashicorp/hcl/hcl/ast"
+	"github.com/hashicorp/hcl/hcl/parser"
+	"github.com/wata727/tflint/config"
+	"github.com/wata727/tflint/evaluator"
+	"github.com/wata727/tflint/issue"
+)
+
+type TestDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateTestDetector() *TestDetector {
+	return &TestDetector{d}
+}
+
+func (d *TestDetector) Detect(issues *[]*issue.Issue) {
+	*issues = append(*issues, &issue.Issue{
+		Type:    "TEST",
+		Message: "this is test method",
+		Line:    1,
+		File:    "",
+	})
+}
+
+func TestDetectByCreatorName(creatorMethod string, src string, c *config.Config, awsClient *config.AwsClient, issues *[]*issue.Issue) {
```

Comment:
__[gometalinter]__
* [golint] exported function TestDetectByCreatorName should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->